README.md

```diff
@@ -1,2 +1,161 @@
+You can then retreive the current status of the current session like that:
+
+```kotlin
+class MainActivity : AppCompatActivity() {
+
+    private val yotiDocScan = YotiDocScan(this)
+
+    override fun onActivityResult(requestCode: Int, resultCode: Int, data: Intent?) {
+        super.onActivityResult(requestCode, resultCode, data)
+        if (requestCode == YOTI_DOCS_REQUEST_CODE) {
+
+            val sessionStatusCode = yotiDocScan.sessionStatusCode
+            val sessionStatusDescription = yotiDocScan.sessionStatusDescription
+
+            result_text.text = "Result code: $sessionStatusCode Message: $sessionStatusDescription"
+        }
+    }
+}
+```
+
+
+### Possible status for the session
+
+| Code              | Message                      | Retry possible for the same session                    |
+| ----------------- | ---------------------------- | ---------------------------------- |
+| 0                 | Result with success          | No                                 |
+| 1000              | No error occurred - the end-user cancelled the session for an unknown reason.          | Yes |
+| 2000              | Unauthorised request (wrong or expired session token).          | Yes |
+| 2001              | Session not found.          | Yes |
+| 2003              | SDK launched without session Token.          | Yes |
+| 2004              | SDK launched without session ID.          | Yes |
+| 3000              | Yoti's services are down or unable to process the request.          | Yes |
+| 3001              | An error occurred during a network request          | Yes |
+| 3002              | User has no network          | Yes |
+| 4000              | The user did not grant permissions to the camera          | Yes |
+| 5000              | No camera.(When user's camera was not found and file upload is not allowed)          | No |
+| 6000              | SDK is out-of-date - please update the SDK to the latest version          | No |
+| 6001              | Unexpected internal error          | No |
+| 6002              | Unexpected document scanning error          | No |
+
+
+## Customisation
+You can customise the appearance of the screens of the SDK by overriding some styles.
+
+### Font Family and Font Colour
+In order to change the font family and the font colour you just need to define the following styles:
+
+```xml
+<style name="Yoti.v3.Font.DisplayBold">
+<style name="Yoti.v3.Font.DisplayRegular">
+<style name="Yoti.v3.Font.TextBold">
+<style name="Yoti.v3.Font.TextRegular">
+<style name="Yoti.v3.Font.TextRegular.RadioButton">
+<style name="Yoti.v3.Font.TextBold.RadioButton">
+```
+
+If you just want to change the colour of the font, then override the following colour:
+
+```xml
+<color name="yotiDocs_colorFont">
+```
+
+### Primary Colours
+In order to change the colours of the different elements of the screens you just need to declare the following colours:
+
+- For the main app branding colour:
```

Comment:
Here we can also add a mention to personalise the font color with `yotiDocs_colorFont`

Reply:
That's been mention in the font section above. 👍